### PR TITLE
GP2 Onboarding - user core details

### PR DESCRIPTION
**Pull request by markcsims**
https://asaphub.atlassian.net/browse/GP2-110

### CODE REVIEW

File: apps/gp2-frontend/src/onboarding/Routes.tsx [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
-        <Route exact path={path}>
-          <OnboardWelcome />
-        </Route>
-      </Switch>
-    </BasicLayout>
+    <Switch>
+      <Route exact path={path}>
+        <Welcome />
+      </Route>
+      <Route exact path={onboarding({}).coreDetails.template}>
```
Comment (jhonny-m) [inline apps/gp2-frontend/src/onboarding/Routes.tsx:17]: in this case it's gonna be the same but I like using the `onboarding({}).coreDetails({}).$` better

File: packages/gp2-components/src/templates/OnboardingCoreDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+});
+const descriptionStyles = css({ paddingBottom: rem(32) });
+const cardStyles = css({ padding: `${rem(32)} 0` });
+const OnboardingCoreDetails: React.FC<OnboardingCoreDetailProps> = ({
+  email,
+  ...headerProps
+}) => (
+  <div css={contentStyles}>
+    <div css={descriptionStyles}>
+      In order to join the platform, we need to capture some core information
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/OnboardingCoreDetails.tsx:77]: shouldn't we wrap the text on text element?

File: packages/gp2-components/src/templates/OnboardingCoreDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      before you start exploring.
+    </div>
+    <UserDetailHeader {...headerProps} edit={'/edit'} />
+    <div css={cardStyles}>
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/OnboardingCoreDetails.tsx:81]: we are adding a lot of padding between elements, can't we just use an gap between them?

File: packages/gp2-components/src/templates/OnboardingWelcome.tsx [hunk 4]
```diff
@@ -70,12 +77,24 @@ const OnboardWelcome: React.FC<Record<string, never>> = () => (
       GP2 members that have also registered.
     </Caption>
     <footer css={footerStyles}>
-      <Button primary>Get Started</Button>
-      <Link buttonStyle href={logout({}).$}>
-        Sign Out
-      </Link>
+      <div css={signOutStyles}>
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/OnboardingWelcome.tsx:80]: this also looks like a gap to me

File: packages/react-components/src/atoms/Link.tsx [hunk 5]
```diff
@@ -64,6 +64,7 @@ interface NormalLinkProps {
   readonly small?: undefined;
   readonly enabled?: undefined;
   readonly noMargin?: undefined;
+  readonly tabletFullWidth?: undefined;
```
Comment (jhonny-m) [inline packages/react-components/src/atoms/Link.tsx:67]: I don't like it being so specific

File: packages/gp2-components/src/organisms/UserDetailHeader.tsx [hunk 6]
```diff
@@ -91,45 +93,55 @@ const UserDetailHeader: React.FC<UserDetailHeaderProps> = ({
   city,
   country,
   positions,
+  edit,
 }) => (
   <header>
-    <BackLink href={backHref} />
+    {backHref && <BackLink href={backHref} />}
```
Comment (markcsims) [inline packages/gp2-components/src/organisms/UserDetailHeader.tsx:99]: Create new component

File: apps/gp2-frontend/src/onboarding/Routes.tsx (hunk 1 above)
Comment (markcsims) [reply apps/gp2-frontend/src/onboarding/Routes.tsx:17]: Done

File: packages/gp2-components/src/templates/OnboardingWelcome.tsx (hunk 4 above)
Comment (markcsims) [reply packages/gp2-components/src/templates/OnboardingWelcome.tsx:80]: done

File: packages/react-components/src/atoms/Link.tsx (hunk 5 above)
Comment (markcsims) [reply packages/react-components/src/atoms/Link.tsx:67]: agreed. It was something I wanted to discuss.

Agreed, changed now to fullWidth

File: packages/gp2-components/src/organisms/UserDetailHeader.tsx (hunk 6 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/UserDetailHeader.tsx:14]: done

File: packages/gp2-components/src/templates/OnboardingCoreDetails.tsx (hunk 2 above)
Comment (markcsims) [reply packages/gp2-components/src/templates/OnboardingCoreDetails.tsx:78]: done

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**